docs/translations/api-docs/date-pickers/date-field-pt.json

```diff
@@ -2,7 +2,7 @@
   "componentDescription": "",
   "propDescriptions": {
     "autoFocus": "If <code>true</code>, the <code>input</code> element is focused during the first mount.",
-    "color": "The color of the component. It supports both default and custom theme colors, which can be added as shown in the <a href=\"https://mui.com/material-ui/customization/palette/#adding-new-colors\">palette customization guide</a>.",
+    "color": "The color of the component. It supports both default and custom theme colors, which can be added as shown in the <a href=\"https://mui.com/material-ui/customization/palette/#custom-colors\">palette customization guide</a>.",
```

Comment:
Not sure where this change comes from but it's not related to this PR :+1:

Reply:
Looking at the updated live docs, you'll find that the previous link is now stale. It was brought to our attention by a failing CI check so we checked the live docs to confirm and then went on to update the link.

Command: yarn docs:link-check